src/sonic-frr-mgmt-framework/templates/bgpd/bgpd.conf.db.nbr_af.j2

```diff
@@ -112,16 +112,12 @@
 {% endif %}
 {# ------- route-map in --------------------------- #}
 {% if 'route_map_in' in n_af_val %}
-{% for map in n_af_val['route_map_in'] %}
-  neighbor {{nbr_name}} route-map {{map}} in
```

Comment:
This code helps to associate multiple route-maps to a neighbor, what issue did you find?

Reply:
This is the second issue listed in https://github.com/sonic-net/sonic-buildimage/issues/20019

In frrcfgd.py, it picks the one input as well

https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py#L1827

Reply:
This means that the data stored in the neighbor_af table is not in list. Refer the below YANG for syntax (use leaf-list)

https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-bgp-common.yang#L385

Reply:
How does frrcfgd.py handle it, not a list, right?

Reply:
No, frrcfgd should handle the value in a list.

Reply:
Ok. I will revert this change and change the original issue 2 to add list handling in frrcfgd.